rewriter/ClassNew.cc

```diff
@@ -40,13 +40,11 @@ vector<ast::ExpressionPtr> ClassNew::run(core::MutableContext ctx, ast::Assign *
         return empty;
     }
 
-    auto recv = ast::cast_tree<ast::UnresolvedConstantLit>(send->recv);
-    if (recv == nullptr) {
+    if (!ast::MK::isRootScopedSyntacticConstant(send->recv, {core::Names::Constants::Class()})) {
```

Comment:
I know we have plenty of them already but `MK` always looks weird for such helpers. I'd prefer them in `Util`:

```
ast::Util::isRootScopedSyntacticConstant
```